manifests/init.pp

```diff
@@ -974,10 +974,14 @@
     }
   }
 
+  if getvar('::ipaddress') and getvar('::ipaddress6') { $host_aliases = [$::hostname, $::ipaddress, $::ipaddress6] }
```

Comment:
why the use of `getvar()` ? you can do the conditionals without that.

Reply:
Nope. Gives 835 examples, 603 failures:
Evaluation Error: Unknown variable: '::ipaddress6'.
And on my IPv6 only system it would fail because the ::ipaddress variable is not defined. Note that the ipaddress and/or ipaddress6 facts are only present when the corresponding addresses are present. If the fact is not defined, then the variable in Puppet itself is also not defined, so you cannot simply use it in an if statement.

Reply:
would you mind adding a short comment to the code with that explanation. The getvar() is not normally used and that would help anyone else reading the code.

Reply:
Done. Now it's just waiting for Travis to complete